jsonpickle/pickler.py

```diff
@@ -35,6 +35,7 @@ def encode(
     separators=None,
     include_properties=False,
     handle_readonly=False,
+        ignore_iterator=False
```

Comment:
This could be an artifact of the github web formatting, but it looks to be indented here at one level past the other options. Could you format the code with black just in case it isn't an artifact?

Reply:
Fixed it myself. Waiting to merge now until discussion on the PR is resolved to a satisfactory extent.